src/components/LightningWallet.vue

```diff
@@ -842,7 +844,26 @@ export default {
       this.send.isSending = false;
     },
     async createInvoice() {
+      // Check that there is enough in the recieve
+      if (!this.maxRecieve || this.recieve.amount > this.maxRecieve) {
+        const toastOptions = {
+          title: "Low incoming balance",
+          autoHideDelay: 3000,
+          variant: "warning",
```

Comment:
Since it's an error:

```suggestion
          variant: "danger",
```